livereveal/main.js

```diff
@@ -348,15 +351,18 @@ function buttonExit() {
 }
 
 function Remover() {
-  $('div#notebook_panel').css('background-color','#ffffff');
-  $('div#header-container').toggle();
+  $('div#site').css("height", "");  
```

Comment:
I see you use this `""` (empty) definitions with the css... I am not a css guy, so... just asking before looking into the net... this "practice" allows you to delete the current css?